openbas-framework/src/main/java/io/openbas/injector_contract/Contract.java

```diff
@@ -52,13 +53,18 @@ public class Contract {
   @JsonProperty("platforms")
   private List<PLATFORM_TYPE> platforms = new ArrayList<>();
 
+  @NotEmpty
+  @JsonProperty("execution_arch")
+  private Payload.PAYLOAD_EXECUTION_ARCH executionArch;
```

Comment:
Why adding executionArch in the contract ?

Reply:
I change! ty!